data/categories.txt

```diff
@@ -216,7 +216,7 @@ pl:4Stacja paliw|3Benzyna|4Paliwo
 pt:3Combustível
 pt-BR:Combustível
 ro:4Benzinărie|Benzină|Gaz
-es:3Gasolinera
+es:3Gasolinera|4Estación de servicio|3gasoil|3diesel|3nafta|4combustible|bencina|3gasolina
```

Comment:
Note that the first, third and last keyword all will be shown/matched when user types 3 chars `gas`. Set a number only for one of them, the most popular/recognizable one, and leave others without numbers, just to match the full word if someone types or pastes them.